cluster-autoscaler/core/scale_down.go

```diff
@@ -207,12 +209,18 @@ func (sd *ScaleDown) UpdateUnneededNodes(
 			glog.V(4).Infof("Node %s is not suitable for removal - utilization too big (%f)", node.Name, utilization)
 			continue
 		}
+		cores, memory, err := getNodeCoresAndMemory(node)
+		if err != nil {
+			glog.Warningf("Error getting node resources: %v", err)
+		}
+		coresTotal = coresTotal + cores
```

Comment:
Move this up.

Reply:
Replaced with constant as discussed offline